cypress/cypress/integration/video.spec.ts

```diff
@@ -0,0 +1,145 @@
+        cy.get("[data-cy=mentor-name]")
+          .should("exist")
+          .contains("Clinton Anderson");
+        cy.get("[data-cy=fave-button]").should("exist");
+      });
+    });
+
+    cy.get("[data-cy=header]").should("have.attr", "data-mentor", "clint");
+    cy.get("[data-cy=topic-2] button").trigger("mouseover").click();
+    cy.get("[data-cy=history-chat]").should("exist");
+
+    cy.get("[data-cy=input-field]").type("user msg 1");
+    cy.get("[data-cy=input-send]").trigger("mouseover").click();
+
+    cy.get("[data-cy=history-chat").within(($hc) => {
+      cy.get("[data-cy=chat-msg-1]").contains("user msg 1");
+    });
+
+    // Mentor Card at the end
+    cy.get("[data-cy=video-container").within(($hc) => {
+      cy.get("[data-cy=mentor-name-card]").should("exist");
+      cy.get("[data-cy=mentorname-faveicon-wrapper]").within(($hc) => {
+        cy.get("[data-cy=mentor-name]")
+          .should("exist")
+          .contains("Clinton Anderson");
+        cy.get("[data-cy=fave-button]").should("exist");
+      });
+    });
+  });
+
+  it("Title Header for one mentor", () => {
+    mockDefaultSetup(cy, {
+      config: {
+        cmi5Enabled: false,
+        mentorsDefault: ["clint"],
+        styleHeaderColor: "#990000",
+        styleHeaderTextColor: "#FFFFFF",
+        styleHeaderLogo: FAKE_STYLE_HEADER_LOGO,
+      },
+      mentorData: [clint],
+    });
+    cy.viewport(1200, 700);
+    cy.intercept(FAKE_STYLE_HEADER_LOGO, { fixture: "uscheader2.png" });
+    cy.intercept("**/questions/?mentor=clint&query=*", {
+      fixture: "response_with_feedback.json",
+    });
+    cy.visit("/");
+
+    cy.get("[data-cy=header]")
+      .within(($h) => {
+        cy.get("img")
+          .should("have.attr", "src")
+          .and("eq", FAKE_STYLE_HEADER_LOGO);
+        cy.get("p").should("have.css", "color", "rgb(255, 255, 255)");
+      })
+      .and("have.css", "background-color", "rgb(153, 0, 0)")
+      .contains("Clinton Anderson: Nuclear Electrician's Mate");
+  });
+
+  it("Title Header for two or more mentors", () => {
+    mockDefaultSetup(cy, {
+      config: {
+        cmi5Enabled: false,
+        mentorsDefault: ["clint", "carlos"],
+        styleHeaderColor: "#990000",
+        styleHeaderTextColor: "#FFFFFF",
+        styleHeaderLogo: FAKE_STYLE_HEADER_LOGO,
+      },
+      mentorData: [clint, carlos],
+    });
+    cy.viewport(1200, 700);
+    cy.intercept(FAKE_STYLE_HEADER_LOGO, { fixture: "uscheader2.png" });
+    cy.intercept("**/questions/?mentor=clint&query=*", {
+      fixture: "response_with_feedback.json",
+    });
+    cy.visit("/");
+
+    cy.get("[data-cy=header]")
+      .within(($h) => {
+        cy.get("img")
+          .should("have.attr", "src")
+          .and("eq", FAKE_STYLE_HEADER_LOGO);
+        cy.get("p").should("have.css", "color", "rgb(255, 255, 255)");
+      })
+      .and("have.css", "background-color", "rgb(153, 0, 0)")
```

Comment:
double check what's correct here, I would have guessed it should be

```typescript
cy.get("[data-cy=header]")
      .within(($h) => {
        cy.get("img")
          .should("have.attr", "src")
          .and("eq", FAKE_STYLE_HEADER_LOGO);
        cy.get("p").should("have.css", "color", "rgb(255, 255, 255)");
      })
      .should("have.css", "background-color", "rgb(153, 0, 0)")
      .and.contains("Mentor Panel");
```

...but I'm not sure

Reply:
`.and` is a function that takes some parameters. It should be:
```
.should("have.css", "background-color", "rgb(153, 0, 0)")
      .contains("Mentor Panel");

https://github.com/mentorpal/mentor-client/pull/84/commits/57173cd0d50f096f56aca8d5fe7de3c8811fa733